modules/core/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ ocv_add_dispatched_file(stat SSE4_2 AVX2)
 ocv_add_dispatched_file(arithm SSE2 SSE4_1 AVX2 VSX3)
 ocv_add_dispatched_file(convert SSE2 AVX2)
 ocv_add_dispatched_file(convert_scale SSE2 AVX2)
+ocv_add_dispatched_file(sum AVX2)
```

Comment:
Why SSE2 dispatching is added for `convert` but omitted for `sum`?

Reply:
Thanks! Added.